### ISSUE TITLE
parameterTree parameter with "dec" option on crashes around zero

### ISSUE THREAD

**Issue by llombard** [pyqtgraph/pyqtgraph#1756]
### Short description
when stepping up or down around zero in a dec spinbox, the step should be some default "minStep", otherwise a fraction of zero is still zero. However, it crashes.

### Code to reproduce
start pyqtgraph "ParameterTree" example
change "DEC stepping" to 0 and step up or down

### Expected behavior
the value should step up or down by a minStep value

### Real behavior
crashes

```
Traceback (most recent call last):
  File "C:\apps\Anaconda3\lib\site-packages\pyqtgraph\widgets\SpinBox.py", line 403, in stepBy
    step = self.opts['minStep']
KeyError: 'minStep'
```

### Tested environment(s)

 * PyQtGraph version: 0.11.0
 * Qt Python binding: 'PyQt5 5.9.2 Qt 5.9.7'
 * Python version: Python 3.8.5 (default, Sep  3 2020, 21:29:08) [MSC v.1916 64 bit (AMD64)]
 * NumPy version: 1.19.2
 * Operating system: windows
 * Installation method: conda

**Comment by ixjlyons** [comment]
Confirmed on master. There's a note about requiring 'minStep' for dec mode in the `SpinBox` docs, but I think it should set a reasonable default as a fallback. Plus `WidgetParameterItem` doesn't forward that option to `SpinBox` for float type parameters.

So, I'd say `SpinBox` needs to set a default 'minStep' if it's not provided (I guess 'step' would be a reasonable default) and `WidgetParameterItem` needs to handle `minStep` for float the float case.

**Comment by ixjlyons** [comment]
Fixed by #1779
